### DISCUSSION TITLE
Live Activities support

### DISCUSSION THREAD

**Discussion by JigSawFr** [LeoKlaus/Paperparrot#170] [Feature Requests]
Hi there,
It can be a nice idea to implement "Live Activities" for paperless. To display activities when paperless is consuming docs!
Have a nice day,

**Comment by LeoKlaus** [comment]
Hey JigSaw,
thanks for the suggestion.
Last time I checked, the consumption progress was only transmitted via Websockets, which do not accept token authentication.

It would surely be possible to implement live activities, but it would be very complicated and (apart from looking pretty cool), I don't think they're much of an improvement over [push notifications](https://paperparrot.me/docs/tutorial-advanced/push-notifications) in terms of informing users of changes.
